.vib/suitecrm/vib-publish.json

```diff
@@ -17,14 +17,57 @@
       ]
     },
     "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/suitecrm"
+        },
+        "runtime_parameters": "c2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0czoKICAgIGh0dHA6IDgwCnN1aXRlY3JtVXNlcm5hbWU6IHVzZXIKc3VpdGVjcm1QYXNzd29yZDogQ29tcGxpY2F0ZWRQYXNzd29yZDEyMyE0CnN1aXRlY3JtU210cEhvc3Q6IHRlc3Rob3N0CnN1aXRlY3JtU210cFBvcnQ6IDkxMjUKc3VpdGVjcm1TbXRwVXNlcjogdGVzdEhvc3RVc2VyCnN1aXRlY3JtSG9zdDogdm13YXJlLXN1aXRlY3JtLm15CnN1aXRlY3JtU210cFBhc3N3b3JkOiB0ZXN0SG9zdFBhc3N3b3Jk",
```

Comment:
We are setting the `suitecrmSmtpPassword` parameter but not using it in our tests, is it necessary for SMTP to work properly or can we do without it?

Reply:
I've tried this and it seems that the parameter is not necessary so I updated the `runtime_parameters`.